lib/glimesh_web/templates/oauth2_provider/authorization/new.html.eex

```diff
@@ -0,0 +1,33 @@
+<div class="container" style="text-align:center;">
+  <h1>Authorize <strong><%= @client.name %></strong> to use your account?</h1>
+
+  <div>
+    <p>This application will be able to:</p>
+    <ul>
+      <%= for scope <- @scopes do %>
+        <li><%= scope %></li>
```

Comment:
This needs to be figured out a way to implement language based control for now it will list the raw values that is feed with the `scopes=` argument in the URL. It will always be a set list that being those set as default_scopes and optional_scopes in the config.exs